api/gloo/enterprise.gloo/v1/auth_config.proto

```diff
@@ -23,6 +23,7 @@ import "google/protobuf/empty.proto";
 message AuthConfigSpec {
 
 
+    reserved 1;
```

Comment:
Here and other `*.proto` files in this PR have a reserved field added, but no field removed. Why is this necessary?